src/frontend/staff-portal/src/app/components/ticket-info/ticket-info.component.ts

```diff
@@ -1,147 +1,575 @@
+      this.initialDisputeValues = this.setViolationTicketFieldsFromJSON(response);
+      this.lastUpdatedDispute = this.initialDisputeValues;
+      this.form.patchValue(this.initialDisputeValues);
+
+      // set violation date and time
+      let violationDate = new Date(response.issuedDate);
+      this.form.get('violationTicket').get('issuedDate').setValue(response.issuedDate);
+      this.form.get('violationTicket').get('violationDate').setValue(this.datePipe.transform(violationDate, "yyyy-MM-dd"));
+      this.form.get('violationTicket').get('violationTime').setValue(this.datePipe.transform(violationDate, "hhmm"));
+
+      // ticket image
+      this.imageToShow = 'data:image/png;base64,' + this.initialDisputeValues.violationTicket.violationTicketImage.image;
+
+      // set disputant detected ocr issues
+      this.flagsForm.get('disputantOcrIssuesDescription').setValue(response.disputantOcrIssuesDescription);
+
+      // set provincial court hearing location in violation ticket subform
+      this.form.get('violationTicket').get('provincialCourtHearingLocation').setValue(
+        this.form.get('provincialCourtHearingLocation').value
+      );
+
+      // set counts 1,2,3 of violation ticket
+      var i = 1;
+      this.initialDisputeValues.violationTicket.violationTicketCounts.forEach(violationTicketCount => {
+
```

Comment:
either remove count1, count2, count3
nor the following
this.form.get('violationTicket').get('violationTicketCount' + violationTicketCount.count).patchValue(violationTicketCount);

Reply:
will do